pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -1274,3 +1274,14 @@ def func(ser):
 
     expected = DataFrame([[1.0]], index=[1])
     tm.assert_frame_equal(res, expected)
+
+
+@pytest.mark.parametrize("dtype", ["int64", "int32", "float64", "float32"])
```

Comment:
can you add Int64, UInt64, Int32, UInt32, Float32, Float64, uint64, uint32 as well here.